pyuvsim/simsetup.py

```diff
@@ -1083,80 +1090,108 @@ def initialize_uvdata_from_params(obs_params):
     return uv_obj, beam_list, beam_dict
 
 
-def initialize_uvdata_from_keywords(yaml_filename=None, antenna_layout_filename=None, array_layout=None, telescope_location=None,
-                                    telescope_name=None, Nfreqs=None, start_freq=None, bandwidth=None, freq_array=None, channel_width=None,
-                                    Ntimes=None, integration_time=None, start_time=None, time_array=None,
-                                    bls=None, antenna_nums=None, antenna_names=None, polarizations=None, no_autos=False,
-                                    redundant_threshold=None, run_check=True, write_files=True, path_out=None, **kwargs):
+def initialize_uvdata_from_keywords(
+        yaml_filename=None, antenna_layout_filepath=None, output_layout_filename=None, array_layout=None,
```

Comment:
I'm realizing that `array_layout` is an ambiguous keyword. Currently, it's only the array layout dictionary, so maybe it should be renamed to `antenna_layout_dict` or something?

Reply:
I don't find it terribly ambiguous -- especially with the docstring specifying it quite precisely. Is there extra information engendered by "array_layout" that isn't actually present in the dict? Possibly just calling it `antpos` or `antenna_pos` is more clear, but `array_layout` also makes sense to me.